cli-wrap/src/main/resources/cli/da_cli_script.py

```diff
@@ -1,25 +1,59 @@
 #!/usr/bin/env python
 import argparse
 import requests
+requests.packages.urllib3.disable_warnings()
 import sys
 import json
 import os
+import getpass
 
 
-da_server = os.getenv('DA_SERVER', "pnc-da-cli.cloud.pnc.devel.engineering.redhat.com/da/rest/v-0.4")
+with open('config.json') as config_file:    
+    config = json.load(config_file)
+da_server = os.getenv('DA_SERVER', config["daServer"])
```

Comment:
verify that da_server is set and not empty and if not, tell user to use configuration or the env variable.